Header header
int64[] ids
float64[] ranges
float64[] angles
